elisp-bytecode.texi

```diff
@@ -1878,62 +2185,79 @@ Make a binding recording buffer, point, and mark.
 @item @verb{|0161|}
 @tab @verb{| 113|}
 @tab @verb{|  set-buffer|}
-@tab Call @code{set-buffer} with one argument.
-@tab @math{-1+1}
+@item @verb{|0162|}
+@tab @verb{| 114|}
+@tab @verb{|save-current-buffer-1|}
+@tab Call @code{save-current-buffer} with one argument.
```

Comment:
Again, I don't think this is best described as a "call" instruction.

Reply:
Ok. Suggest something.